index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+   <!-- <div class="bg-image" 
+     style="background-image: url('https://mdbootstrap.com/img/Photos/Others/images/76.jpg');
+            height: 100vh">
+-->
+    <div class="container-md bg-info bg-opacity-25 ">
+    <h1 id="title">This is the title</h1>
+
+    <p id="description">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Esse placeat vel cupiditate libero temporibus officia cum cumque debitis, quaerat possimus! Quia voluptates, aliquam voluptate molestiae natus magni debitis error quis!</p>
+    
+    <form action="" id="survey-form">
+      <div class="mb-3">
+      <label for="name" id="name-label">Name:</label>
+      <input class="form-control" type="text" name="name" id="name" placeholder="Put your name here!" required>
+      </div>
+
+      <div class="mb-3">
+      <label for="email" id="email-label">Email:</label>
+      <input class="form-control" type="email" name="email" id="email" placeholder="Put your email here!" required>
+      </div>
+
+      <div class="mb-3">
+      <label for="number" id="number-label">Number</label>
+      <input class="form-control" type="number" name="number" id="number" min="0" max="100" placeholder="Put a number between 0 and 100!">
+      </div>
+
+      <div class="mb-3">
+      <label for="dropdown" id="dropdown-label">Which option best describes your current role?</label>
+      <select class="form-select" name="dropdown" id="dropdown">
+        <option value="Student">Student</option>
+        <option value="Full_time">Full time job</option>
+      </select>
+      </div>
+      <label for="radio1" id="radio1-label">Select an option</label>
+      <div class="mb-3 form-check">
+      
+      <input class="form-check-input" type="radio" name="radio1" id="radio1" value="value1">value1
+      </div>
+      <div class="mb-3 form-check">
+      <input class="form-check-input" type="radio" name="radio1" id="radio2" value="value2">value2
+      </div>
+      
+
```

Comment:
:arrow_forward: Kindly deletes excessive blank lines in your code.

:eyes:  Remember the "Best practices" followed in Microverse:
**_Be careful with blank lines and indentation._**
- Do not add blank lines, spaces, or indentations without a reason.
- For readability, add blank lines to separate large or logical code blocks.
- For readability, add two spaces of indentation. Do not use the tab key.
- on some code editors, you can set the tab button to do an X amount of spaces.